tasks/deploy.yml

```diff
@@ -2,11 +2,7 @@
 
 - name: Set Fact nex_app_shared_mounts
   set_fact:
-    nex_app_shared_mounts:
-      - var
-      - app/etc
-      - pub/media
-      - pub/static
+    nex_app_shared_mounts="{{ magento_shared_mounts }}"
```

Comment:
It feels kind of gross for the magneto role to be setting nex_app_shared_mounts, however, the alternative seemed worse, which was having every app role do the sync'ing and mounting of the shared dirs itself.